app/main.css

```diff
@@ -60,3 +60,13 @@ select {
   --theme-font-size-large: 20px;
   --theme-toolbar-height: 63px;
 }
+
+:global #app {
+  height: 100%;
+  display: flex;
```

Comment:
Why do we need to use flexbox on the root element? [This](https://jakearchibald.com/2014/dont-use-flexbox-for-page-layout/) keeps popping in my head.

Reply:
so that the `<main>` can get `flex: 1` and stretch vertically to the bottom of the screen (I'm happy to have alternative solutions)